packages/react-native-sdk/src/components/call/CallTopView/CallTopView.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useState } from 'react';
+import {
+  View,
+  StyleSheet,
+  Text,
+  StyleProp,
+  ViewStyle,
+  Pressable,
+} from 'react-native';
+import { ParticipantsInfoBadge } from './ParticipantsInfoBadge';
+import { theme } from '../../../theme';
+import { Back } from '../../../icons/Back';
+import { Z_INDEX } from '../../../constants';
+import { TopViewBackground } from '../../../icons';
+
+export type CallTopViewProps = {
+  /**
+   * Handler to be called when the back button is pressed in the CallTopView.
+   * @returns void
+   */
+  onBackPressed?: () => void;
+  /**
+   * Handler to be called when the Participant icon is pressed in the CallTopView.
+   * @returns
+   */
+  onParticipantInfoPress?: () => void;
+  /**
+   * Title to be rendered at the center of the Header.
+   */
+  title?: string;
+  /**
+   * Style to override the container of the CallTopView.
+   */
+  style?: StyleProp<ViewStyle>;
+};
+
+export const CallTopView = ({
+  onBackPressed,
+  onParticipantInfoPress,
+  title,
+  style,
+}: CallTopViewProps) => {
+  const [headerHeight, setHeaderHeight] = useState<number>(0);
+
+  const onLayout: React.ComponentProps<typeof View>['onLayout'] = (event) => {
+    const { height } = event.nativeEvent.layout;
+    setHeaderHeight(height);
+  };
+
+  return (
+    <>
+      {/* Component for the background of the CallTopView. Since it has a Linear Gradient, an SVG is used to render it. */}
+      <TopViewBackground
+        height={headerHeight}
+        width={'100%'}
+        style={[styles.background, style]}
```

Comment:
```suggestion
        style={[styles.background]}
```

passing style here would cause unnecessary bug, we already use the height from layout

Reply:
This was added for devices with top insets, in that case, the top margin needs to be adjusted.

Reply:
Done ✅